ubuntu/R/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM databricksruntime/minimal:latest
 
 # Ubuntu 16.04.3 LTS installs R version 3.2.3 by default. This is fairly out dated.
-# We add RStudio's debian source to install the latest r-base version (3.4.1)
+# We add RStudio's debian source to install the latest r-base version (3.6.0)
```

Comment:
I think this still installs an outdated version - `3.2.3`.

Reply:
Ubuntu installs 3.2.3 by default, the idea of this code block is upgradings by installing a newer version of r-base. On my testing this installed v3.6.0, you can check by running R in the container and then enter 'version'

Reply:
From a fresh build with your changes.
```
> version
               _
platform       x86_64-pc-linux-gnu
arch           x86_64
os             linux-gnu
system         x86_64, linux-gnu
status
major          3
minor          2.3
year           2015
month          12
day            10
svn rev        69752
language       R
version.string R version 3.2.3 (2015-12-10)
nickname       Wooden Christmas-Tree
```

Still investigating

Reply:
Ok figured it out. Since I can't push to your PR I'll make a followup.

Reply:
ok so I failed to translate this correctly from my working version in a different repo, will update shortly